doc/api/net.md

```diff
@@ -501,7 +507,30 @@ added: v0.5.3
 
 The amount of bytes sent.
 
-### socket.connect(options[, connectListener])
+### socket.connect()
+
+Initiate the connection on a given socket.
+
+Normally this method is not needed, the socket should be created and
+opened with [`net.createConnection()`][]. Use this only if you are
+implementing a custom Socket.
+
+Possible signatures:
+
+* [socket.connect(options[, connectListener])][`socket.connect(options)`]
+* [socket.connect(path[, connectListener])][`socket.connect(path)`]
+  for connections to UNIX domain sockets or Windows named pipes.
```

Comment:
https://msdn.microsoft.com/en-us/library/windows/desktop/aa365590(v=vs.85).aspx

So, it looks like named pipes can be used remotely. But, would you object to me describing something as a "local file" because when running an NFS or SMB file server, local files can be accessed remotely? IMO, the pipe is local, even though there is a network protocol layer that supports remote access to the local pipe.